### PR TITLE
SmthGrey audio updates

### PR DESCRIPTION
**Pull request by NoSloppy**
Added boot.wav.
Added endlock made from lock + SmthGrey clash04.
Normalized hum to -5db and high passed hum @100hz - 48db slope.
out.wav Normalized to 0dB
Edited lock to have less monophonic-like hum mixed in.

### CODE REVIEW

**Comment by NoSloppy** [comment]
Deleted highpassed version hum, replaced with normalized only version.

**Comment by profezzorn** [comment]
Needs readme updates for added files.

File: SmthGrey/config.ini [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# SmthGrey config.ini
+
+# For full descriptions of what each of these entries does,
+# along with the default values for each, see the Wiki page here:
+# https://github.com/profezzorn/ProffieOS/wiki/Sound-Font-config.ini-Configuration
+
+# humStart default = 100
+humStart=751
+volHum=15
```
Comment (profezzorn) [inline SmthGrey/config.ini]: I think we should comment out all the variables below which have default values.
Makes it easy to see which ones actually do something.
We can also add a comment about it, something like:

All possible config variables listed below, but variables with default values are commented out.

File: SmthGrey/license.txt [hunk 2]
```diff
@@ -1 +1,11 @@
-These files are free to use, modify, and redistribute without attribution or any other restrictions.
\ No newline at end of file
+These files are free to use, modify, and redistribute without attribution or any other restrictions.
+
+Modifications by Brian Conner 2022:
+Detailed changes can be found in this font's ReadMe.txt.
+Any additional sounds sourced from creative-commons licensed freesound.org.
```
Comment (profezzorn) [inline SmthGrey/license.txt]: Please specify which files.

File: SmthGrey/config.ini (hunk 1 above)
Comment (NoSloppy) [reply SmthGrey/config.ini]: Done.

File: SmthGrey/license.txt (hunk 2 above)
Comment (NoSloppy) [reply SmthGrey/license.txt]: Done.

**Pull request merged**